internal/device/dryRun.go

```diff
@@ -43,3 +43,9 @@ func (b *DryRunBMC) SetBootDevice(_ context.Context, _ string, _, _ bool) error
 func (b *DryRunBMC) PowerCycleBMC(_ context.Context) error {
 	return nil
 }
+
+// HostBooted reports whether or not the device has booted the host OS
+func (b* DryRunBMC) HostBooted(_ context.Context) (bool, error) {
+	// FIXME: is this adequate or should we differentiate based on b.state?
```

Comment:
responding to the FIXME:

nah, should be fine. Its just dummy data

Reply:
Also, if this is returning `(bool, error)` and the bool is just "did it boot successfully?" you can shortcut with just returning an error and having the caller test that `err == nil` (welcome to Golang).

Reply:
Ack, thanks -- I'll drop the comment.

Reply:
> Also, if this is returning (bool, error) and the bool is just "did it boot successfully?" you can shortcut with just returning an error and having the caller test that err == nil (welcome to Golang).

Hmm, that seems like a bit of an error conflation to me?  I suppose in the context of what we're doing with the bool here perhaps we don't care too much about the distinction, but treating "we successfully queried the BMC but the host hasn't (yet) fully booted" as an error strikes me as odd, basically...

Reply:
In that case, maybe we're describing something that's not strictly `bool` in the first return value? Or is it strictly binary?